### ISSUE TITLE
v.3.0 does not support combined temp/hum sensor

### ISSUE THREAD

**Issue by odelma** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#628]
Hello,

I went with the option to update existing devices (I know, not recommended). Overall most devices seem to work and there are also a bunch of new ones. But few issues as well. WSDCGQ01LM Temp/hum sensor stopped reporting under the combined sensor (temp/hum). It was still updating the individual temp and hum sensor. I then tried first re-pairing, second I removed the existing temp/hum device and hoped it would be recreated but it was not. Third, I removed all devices aasociated with this sensor, removed it from zigbee network and added again. Rest of the devices are recreated but not the combiner temp/hum sensor. What should I do to make it alive again (in this case to be created again), is there a way for this plugin to force the (re-) creation of Domoticz devices?

The second issue is with IKEA Kadjril roller blinds. It seems up and down are inverted. This is ok as such, I can just switch them to type "blinds percentage inverted" but it seems that both 0% and 100% will now roll them down. 1% takes them almost up, then 0% again down. On/Off works but in scenes you have to set the percentage and thus scenes don't work anymore. Should I also try to re-create these devices? If so, how should I proceed?

Thanks for the great plugin! :)

**Comment by Belaial** [comment]
Hi,

Regarding your issue with combined temp/hum sensor not reporting, I see the same behavior on another type of sensor, Sonoff SNZB-02, reported in ticket I linked below.

I have not tried yet to downgrade the plugin but I think / guess that would solve it since I have another installation where combined temp/hum still works, in that installation I have not upgraded the plugin after I saw this issue in my first installation.

https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues/623

**Comment by odelma** [comment]
Oh, I did not notice your issue but indeed it looks to be very much similar that what I am experiencing. The main reason I updated was that I was not able to add my newest device due to compatibility and decided this is the only way to go to keep the system future proof.

I guess I can live with the separate temp and hum sensor but of course it would be nice to find out what is causing this and possibly help to get it fixed.

**Comment by BRSS73** [comment]
You can use the following workaround :
https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues/621#issuecomment-817170817

**Comment by odelma** [comment]
Thanks, this is great - and there's even a PR for this now. I wonder if everything is ok with @stas-demydiuk as it seems he has not been active here for a while. Hope everything is ok. If Stas is busy with other activities, do you know if anyone else can merge PR's in this repo?

**Comment by stas-demydiuk** [comment]
Added support for combined devices back, please update the plugin and check

**Comment by salopette** [comment]
2021-05-03 14:24:00.624 Error: EventSystem: TEMP_HUM_BARO missing values : ID=11188, sValue=21.2;48.08;0
2021-05-03 14:24:00.624 Error: EventSystem: TEMP_HUM_BARO missing values : ID=11313, sValue=14.3;44.34;0
2021-05-03 14:24:14.773 EventSystem: Event triggered: Flurlicht_2
2021-05-03 14:24:14.772 Error: EventSystem: TEMP_HUM_BARO missing values : ID=11188, sValue=21.2;48.08;0
2021-05-03 14:24:14.772 Error: EventSystem: TEMP_HUM_BARO missing values : ID=11313, sValue=14.3;44.34;0
2021-05-03 14:24:21.047 EventSystem: Event triggered: Nachtlicht Küche_2
2021-05-03 14:24:21.046 Error: EventSystem: TEMP_HUM_BARO missing values : ID=11188, sValue=21.2;48.08;0
2021-05-03 14:24:21.046 Error: EventSystem: TEMP_HUM_BARO missing values : ID=11313, sValue=14.3;44.34;0

**Comment by salopette** [comment]
After deleting the device from Domoticz, it is back without a falter!
Everything OK!

**Comment by Belaial** [comment]
Updated plugin with "git pull" restarted Domoticz and SNZB-02 combined temp / hum started to work instantly! Great work!

However my

LED1732G11 - TRADFRI LED bulb E27 1000 lumen, dimmable, white spectrum, opal white
LED1733G7 - TRADFRI LED bulb E14 600 lumen, dimmable, white spectrum, opal white

IKEA lamps stopped working after the update, if I try and turn them on via the "xxxxxxx_light" device that I had crated before I get this error in the Domoticz log,

`2021-05-03 19:30:06.773 Status: User: Admin initiated a switch command (9250/Taklampa 2/On)
2021-05-03 19:30:06.810 Error: (Zigbee2MQTTplugin) 'onCommand' failed 'AttributeError':''CTLight' object has no attribute 'feature''.
2021-05-03 19:30:06.810 Error: (Zigbee2MQTTplugin) ----> Line 301 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onCommand
2021-05-03 19:30:06.810 Error: (Zigbee2MQTTplugin) ----> Line 104 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onCommand
2021-05-03 19:30:06.810 Error: (Zigbee2MQTTplugin) ----> Line 75 in '/home/pi/domoticz/plugins/zigbee2mqtt/devices_manager.py', function handle_command
2021-05-03 19:30:06.810 Error: (Zigbee2MQTTplugin) ----> Line 279 in '/home/pi/domoticz/plugins/zigbee2mqtt/adapter.py', function handle_command`

If there is anything I can do to help test it let me know!
